modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,231 @@
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: tempo-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.. Check the Operator status.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-tempo-operator
+----
+
+. Run `oc apply -f` for a YAML file or standard input to create a Secret for your object storage bucket:
++
+[source,terminal]
+----
+$ oc apply -f <secret_file>.yaml
+----
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Run `oc new-project` to create an OpenShift project of your choice for the *TempoStack* instance that you will create in the next step.
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc new-project <openshift_project_of_tempostack_instance>
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: <openshift_project_of_tempostack_instance>
+EOF
+----
+====
+
+. Create a *TempoStack* instance in the OpenShift project that you created for the *TempoStack* instance in the previous step.
++
+NOTE: You can create multiple *TempoStack* instances in separate projects on the same cluster.
++
+.. Customize the `TempoStack` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <openshift_project_of_tempostack_instance>
+spec:
+  storageSize: 1Gi
+  storage:
+      secret:
+          name: <secret-name> <1>
+          type: <secret-provider> <2>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+<1> The value of the `name` in the `metadata` of the Secret.
+<2> The accepted values are `azure` for Azure Blob Storage, `gcs` for Google Cloud Storage, and `s3` for Amazon S3 or MinIO or Red Hat OpenShift Data Foundation.
++
+.TempoStack CR for AWS S3 and MinIO storage
+====
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+  namespace: openshift_project_of_tempostack_instance
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: minio-test
+      type: s3
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+The stack deployed in this example is configured to receive Jaeger Thrift over HTTP and OpenTelemetry Protocol (OTLP), which permits visualizing the data with the Jaeger UI.
+====
+
+.. Apply the customized CR.
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<TempoStack_custom_resource>
+EOF
+----
+
+
+.Verification
```

Comment:
Some command steps under this are missing `[source,terminal]`, all are missing "by running the following command:"

Reply:
Thank you for spotting. Fixed.